pkg/scheduler/internal/cache/cache.go

```diff
@@ -708,6 +710,24 @@ func (cache *cacheImpl) addNodeImageStates(node *v1.Node, nodeInfo *framework.No
 	nodeInfo.ImageStates = newSum
 }
 
+// updateNodeImageStates update node record from image entries having the node
+// in imageStates cache when receive node add/update/delete events.
+func (cache *schedulerCache) updateNodeImageStates(newNode *v1.Node) {
```

Comment:
where is `schedulerCache`? this doesn't seems like compile-able.

Reply:
sorry，i cherry pick a commit base v1.23.6，now fix with cacheImpl struct